modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ObjectDetails.scss

```diff
@@ -19,10 +19,11 @@
 .lfr__object-web-edit-object-details-external-data-source-container {
 	display: grid;
 	position: relative;
-}
 
-.lfr__object-web-edit-object-details-external-data-source-container-beta {
-	left: 200px;
-	margin-top: 7px;
-	position: absolute;
+	&-beta {
+		left: 200px;
+		margin-top: 7px;
+		position: absolute;
+		top: -57px;
```

Comment:
Hey @Artartur, functionally your code is correct and achieving the expected behavior. However, semantically it does not comply with the structure. 

If your goal was to render the beta button next to the panel title, we could have made this change in `EditObjectDetails.tsx` instead use this `top:-57px`.

You should be able to achieve the expected behavior by modifying the way things are done here:

```
					{Liferay.FeatureFlags['LPS-135430'] && (
						<ClayPanel
							collapsable
							defaultExpanded
							displayTitle={
								Liferay.Language.get(
									'external-data-source'
								)}
							displayType="unstyled"
						>
							<ClayPanel.Body>
								<div className="lfr__object-web-edit-object-details-external-data-source-container">
									<ExternalDataSourceContainer
										errors={errors}
										setValues={setValues}
										storageTypes={storageTypes}
										values={values}
									/>

									<div className="lfr__object-web-edit-object-details-external-data-source-container-beta">
										{values.storageType ===
											'salesforce' && <BetaButton />}
									</div>
								</div>
							</ClayPanel.Body>
						</ClayPanel>
					)}
```
[See ](https://clayui.com/docs/components/panel.html)